src/plugins/flexibleLayout/utils/frame.js

```diff
@@ -0,0 +1,9 @@
+class Frame {
+    constructor(domainObject, height, cssClass) {
```

Comment:
We want to avoid inserting classes inside domain objects-- this should be a factory function which returns a new javascript object.